actris/index.html

```diff
@@ -26,7 +26,7 @@
 	  <p class="lead">
 	  Session Types meet Separation Logic</p>
         <p>
-          <a href="https://gitlab.mpi-sws.org/iris/actris" class="btn btn-light border" role="button">Download the Coq source code</a>
+          <a href="https://gitlab.mpi-sws.org/iris/actris" class="btn btn-light border" role="button">Download the Rocq source code</a>
```

Comment:
Arguably this is confusing both before and after since it's not the source code of Rocq that is being referenced here... the usual term is more like "Rocq formalization" or "Rocq development".